package.json

```diff
@@ -16,16 +16,18 @@
     "typescript"
   ],
   "scripts": {
-    "test": "xo && nyc ava"
+    "test": "del-cli test/fixtures/compiled && xo && nyc ava"
```

Comment:
Can we do this deletion in a `before()` instead?

Reply:
sure thing